plot_fluxnet.qmd

```diff
@@ -173,24 +219,25 @@ Null values in FLUXNET are indicated by some variation of -9999 (-9999.x, where
 
 ```{r}
 #| warning: false
-cols_to_check <- c("GPP_NT_VUT_REF", "RECO_NT_VUT_REF", "NEE_VUT_REF", "LE_F_MDS", "H_F_MDS")
+# Replace all numeric values < -9000 with NA
+multiple_sites_annual <- multiple_sites_annual |>
+  mutate(across(where(is.numeric), function(x) if_else(x < -9000, NA, x)))
 
-multiple_sites_annual_nulls <- multiple_sites_annual %>% 
-  select(site, TIMESTAMP, cols_to_check) %>% 
-  filter(if_any(cols_to_check, ~ . < -9000))
+multiple_sites_daily <- multiple_sites_daily |>
+  mutate(across(where(is.numeric), function(x) if_else(x < -9000, NA, x)))
```

Comment:
Is there any reason why we shouldn't assume _all_ -9999s are `NA`s?  Are there columns which would reasonably have values < -9000?